linera-client/src/benchmark.rs

```diff
@@ -37,6 +37,16 @@ use crate::chain_listener::{ChainListener, ClientContext};
 const PROXY_LATENCY_P99_THRESHOLD: f64 = 400.0;
 const LATENCY_METRIC_PREFIX: &str = "linera_proxy_request_latency";
 
+/// Parameters for generating operations on the fly during benchmark execution
+pub struct BenchmarkOperationParams<Env: Environment> {
+    /// The chain client for this benchmark task
+    pub chain_client: ChainClient<Env>,
```

Comment:
This isn't really a "parameter"; maybe `BenchmarkOperationParams` isn't really worth it as a type?